### ISSUE TITLE
Header Styles Being Overwritten by Docsify Styles

### ISSUE THREAD

**Issue by fenriskiba** [fenriskiba/drawcsify#1]
The Docsify CSS has styles for `h1` tags, which impact any headers in the generated Drawio `svg`. As a result, a diagram that should look like this

![Simple Draw.io diagram with a visible heading in a text box.](https://github.com/user-attachments/assets/dcddbd17-fed5-405c-855b-fd0c61e204c6)

Ends up looking like this:
![The same diagram as above, except the heading in the diagram is not visible and the text below it is pushed off the screen as a result of color and size changes.](https://github.com/user-attachments/assets/503264a4-2f08-482a-9375-43aa26505be1)

**Comment by fenriskiba** [comment]
Potentially Relevant Issue: https://github.com/docsifyjs/docsify/issues/131

**Comment by rantoniuk** [comment]
Upstream issue is closed as fixed and I don't see that issue on my test diagram - I guess this can be closed.

**Comment by fenriskiba** [comment]
The upstream issue was closed before I started this, but it was closed without any changes made. I'm not particularly good with CSS, and didn't have a lot of time to look into it, so I'm also not 100% sure it was related. It was more just one avenue to go down in case I did find some time.

This is also still happening in my example: https://fenriskiba.github.io/drawcsify/#/?id=side-by-side

![Screenshot of the example page linked above.](https://github.com/user-attachments/assets/20c68a96-5601-4a7a-9c44-7818870436ac)

**Comment by sy-records** [comment]
It is defined in the theme.
![Image](https://github.com/user-attachments/assets/b26b5166-bbc8-40c4-abcd-cee09f5f6ac7)

**Comment by fenriskiba** [comment]
I know where the styles are coming from, the problem is that I'm having trouble finding a way to ignore them in the context of the `svg` generated by the Drawio viewer.

**Comment by rantoniuk** [comment]
> I know where the styles are coming from, the problem is that I'm having trouble finding a way to ignore them in the context of the svg generated by the Drawio viewer.

I don't think you want to _ignore_ them but to _override_ them for `.mxgraph`-classed div with more specific CSS - otherwise how would the browser know how do you expect this to look like when rendered out by the browser? 

For example, by adding something along below lines, the style would revert to browser defaults:

```css
.markdown-section svg h1 {
  margin: revert;
  padding: revert;
  border-width: revert;
  border-style: revert;
  border-color: revert;
  font-family: revert;
  font-size: revert;
  font-weight: revert;
  line-height: revert;
  color: revert;
}
```

I think additional SCSS is missing with this plugin to style the content accordingly - or experimenting with the [SVG export settings](https://www.drawio.com/doc/faq/export-to-svg) Some related stuff is also in the [Troubleshooting section there](https://www.drawio.com/doc/faq/export-to-svg).

**Comment by fenriskiba** [comment]
That's the problem is that I've tried a lot of those reverts and similar tags, but never gotten anywhere. For some reason they never worked for me.

If I get some time, I'll try what you have there again, but iirc, there is something themeable does that goes through and invalidates those styles at render time and pushes it's own again.

**Comment by fenriskiba** [comment]
In theory, what I currently have there should revert all styles, but evidently it isn't working in the example I posted above. https://github.com/fenriskiba/drawcsify/blob/main/drawcsify.js#L54-L62
